services/audio_library.py

```diff
@@ -0,0 +1,175 @@
+import asyncio
+import threading
+from os import path, listdir
+
+from api.interface import AudioFile, AudioFileConfig
+from services.audio_player import AudioPlayer
+from services.file import get_writable_dir
+
+class AudioLibrary:
+    def __init__(
+        self,
+        callback_playback_started: callable|None = None,
```

Comment:
![CleanShot 2024-08-11 at 14 24 06@2x](https://github.com/user-attachments/assets/8d786bf7-7307-4f71-9a12-4c38f5655568)

This must be callable[TYPE, None]

Reply:
fixed